components/EmotionRecordsList.js

```diff
@@ -112,9 +111,11 @@ export default function EmotionRecordsList({
 
   return (
     <>
-      {emotionEntries.length === 0 && <StyledP>No highlighted entries</StyledP>}
+      {shownEntries.length === 0 && (
+        <StyledTextMessage>Sorry, nothing found</StyledTextMessage>
+      )}
       <StyledList>
-        {emotionEntries.map(
+        {shownEntries?.map(
```

Comment:
As you can see in the element section of the dev-tools, this renders a `<ul>` with `<div>` elements inside instead of `<li>`'s. Your app renders the `<li>` inside the `<div>`.